airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py

```diff
@@ -64,6 +64,12 @@ class FileBasedStreamConfig(BaseModel):
         description="When enabled, syncs will not validate or structure records against the stream's schema.",
         default=False,
     )
+    resent_n_files_to_read_for_schema_discovery: Optional[int] = Field(
```

Comment:
i think this is a small typo. Instead of `resent` it should be `recent`. Resent would imply we're resending something

Reply:
updated